src/manifests.py

```diff
@@ -89,6 +117,7 @@ def get_resource_list(self, check=True):
     def config(self):
         return {
             "image-registry": self.charm_config["image-registry"],
+            "control-node-taints": {},
```

Comment:
Probably I'm missing something here. But, what's the purpose of this config option?

Reply:
I think a copy/paste mistake.

```suggestion
```